spec/controllers/transcriptions_controller_spec.rb

```diff
@@ -108,6 +110,39 @@
           expect(json_data.size).to eq(2)
         end
       end
+
+      it 'filters by updated by' do
+        get :index, params: { filter: { updated_by_eq: another_transcription.updated_by } }
+        expect(json_data.size).to eq(1)
+        expect(json_data.first).to have_id(another_transcription.id.to_s)
+      end
+
+      it 'filters by updated at' do
+        Timecop.travel(Time.now + 1.hour) do
```

Comment:
This is cool, but is loading a whole new gem to do this better/easier than setting `transcription.updated_at` manually? We're not testing that updating the transcription refreshes `updated_at` or `updated_by`, the spec is for the order of the output.

Reply:
I tried setting `updated_at` manually, but it disregarded what I set manually, since it appears that Rails overrides anything set for `updated_at` with the current datetime when it saves the object to the db. Believe me, that was the first thing I tried 😅

Reply:
Couple ways around this. First is setting `ActiveRecord::Base.record_timestamps = false` before updating the record. Second is using `update_column` instead, which updates the db column directly and does not update timestamps, does not run callbacks, does not pass go, collects nothing. Remember this amazing chart?! https://davidverhasselt.com/set-attributes-in-activerecord/

If you're using it somewhere else for a different, un-get-aroundable reason then this is cool, otherwise it seems like overkill.

Reply:
YES the amazing chart!!! Awesome. Let me play around with this.

Reply:
@zwolf - using `update_column` worked magically. So good. Goodbye timecop 👋